assets/js/atomic/blocks/product-elements/image/constants.tsx

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { image, Icon } from '@wordpress/icons';
+
+export const BLOCK_TITLE: string = __(
+	'Product Image',
+	'woo-gutenberg-products-block'
+);
+export const BLOCK_ICON: JSX.Element = (
+	<Icon icon={ image } className="wc-block-editor-components-block-icon" />
+);
+export const BLOCK_DESCRIPTION: string = __(
```

Comment:
What is the consensus of type inference? I think we are removing those, right?

Reply:
We haven't come to a conclusion yet, but all other sections explicitly mention the basic types. That's why I decided to follow this pattern here.